server/TracyView.cpp

```diff
@@ -16561,8 +16561,12 @@ uint32_t View::GetRawZoneColor( const ZoneEvent& ev, uint64_t thread, int depth
 {
     const auto sl = ev.SrcLoc();
     const auto& srcloc = m_worker.GetSourceLocation( sl );
-    const auto color = srcloc.color;
-    if( color != 0 && !m_vd.forceColors ) return color | 0xFF000000;
+    if( !m_vd.forceColors ) {
+      const auto custom_color = m_worker.GetZoneExtra( ev ).color;
+      if( custom_color != 0 ) return custom_color | 0xFF000000;
+      const auto color = srcloc.color;
+      if( color != 0 ) return color | 0xFF000000;
+    }
```

Comment:
Formatting.

Reply:
sorry not seeing it - what would you change? (I tried to match the existing formatting)

Reply:
Move the first { brace to a new line and use 4 spaces to indent instead of 2.

Reply:
ahh nevermind 4 space indent - got it!